pandas/tests/resample/test_base.py

```diff
@@ -195,35 +195,6 @@ def test_resample_empty_dtypes(index, dtype, resample_method):
         pass
 
 
-@all_ts
```

Comment:
can you leave all of these tests. I am ok with either:
- converting to use offset / origin as appopriate, or simply catching the FutureWarning

Reply:
They have been moved into `test_deprecated.py`.
When you told me to remove some deprecated tests I moved them instead into `test_deprecated.py`: https://github.com/pandas-dev/pandas/pull/31809#discussion_r379711472